uvm/servlets/admin/config/network/view/Hostname.js

```diff
@@ -95,9 +98,19 @@ Ext.define('Ung.config.network.view.Hostname', {
             bind: '{settings.dynamicDnsServiceUsername}'
         }, {
             xtype: 'textfield',
-            fieldLabel: 'Password'.t(),
-            bind: '{settings.dynamicDnsServicePassword}',
-            inputType: 'password'
+            fieldLabel: 'Password or API Token'.t(),
+            inputType: 'password',
+            bind: '{settings.dynamicDnsServicePassword}'
+        }, {
+            xtype: 'textfield',
+            fieldLabel: 'Zone'.t(),
+            disabled: true,
+            hidden: true,
+            bind: {
+                value: '{settings.dynamicDnsServiceZone}',
+                disabled: '{settings.dynamicDnsServiceName != "cloudflare" ? true : false}',
```

Comment:
it should be enough just
`{settings.dynamicDnsServiceName !== "cloudflare"}`
without `? true : false`